doc/configuration.rst

```diff
@@ -670,6 +670,13 @@ you can reuse with your own values.
 	[gpg]
 		home = "/home/myuser/.gnupg/"
 		keyid = "E60892BB9BD89A69F759A1A0A3D652173B763E8F"
+        [targets]
+                macro = allonline:status='online'
+                macro = idleandonline:status='online' OR status='idle'
+
+The targets section is optional and provides the ability to specify
+short forms of your own targeting strings. In the example above, 
+`allonline` or `idleandonline` could be used as target arguments.
```

Comment:
Could you add a note to the command_line and console docs too? I'm afraid this configuration doc is already too large for anyone to find anything in it.